src/devices/yandex.ts

```diff
@@ -431,4 +436,48 @@ export const definitions: DefinitionWithExtend[] = [
             }),
         ],
     },
+    {
+        zigbeeModel: ["YNDX-00530"],
+        model: "YNDX_00530",
+        vendor: "Yandex",
+        description: "Dimmer",
+        extend: [
+            YandexCluster(),
+            m.light({
+                effect: true,
+                powerOnBehavior: true,
+                configureReporting: true,
+            }),
+            m.lightingBallast(),
+            binaryWithSetCommand({
+                name: "led_indicator",
+                cluster: "manuSpecificYandex",
+                attribute: "ledIndicator",
+                valueOn: ["ON", 1],
+                valueOff: ["OFF", 0],
+                setCommand: "ledIndicator",
+                zigbeeCommandOptions: {manufacturerCode},
+                description: "Led indicator",
+                entityCategory: "config",
+            }),
+            enumLookupWithSetCommand({
+                name: "button_mode",
+                cluster: "manuSpecificYandex",
+                attribute: "buttonMode",
+                setCommand: "buttonMode",
+                zigbeeCommandOptions: {manufacturerCode},
+                description: "Dimmer button mode",
+                lookup: {
+                    general: 0x00,
+                    alternative: 0x01,
+                },
+                entityCategory: "config",
+            }),
+            m.reportAttribute({
```

Comment:
Is this not already done with:

```js
m.light({
                effect: true,
                powerOnBehavior: true,
                configureReporting: true,
            }),
```

Reply:
No. The default minimum period in the report is 10 seconds, and here I change it to the minimum.

Reply:
If I add reporting parameters for each attribute to m.light, the code will grow. Therefore, I decided to make a universal reporting that can be used in various cases.

Reply:
I think it is nicer to put it in `m.light`, in this was you don't need to look in multiple places for the config (and also the execution order is not guaranteed)

Reply:
ok